chain/jsonrpc/res/network_info.js

```diff
@@ -38,10 +38,28 @@ function computeTraffic(bytes_received, bytes_sent) {
     return "⬇ " + convertBps(bytes_received) + "<br>⬆ " + convertBps(bytes_sent);
 }
 
-function add_debug_port_link(peer_addr) {
+function add_debug_port_link(peer_network_addr) {
+    // Each node running in a machine is assigned ports 24567 + peer_num and 3030 + peer_num, whereby peer_num is a whole number
+    // peer_rpc_address is not shared between peer nodes. Hence, it cannot be programmatically fetched.
+    // https://github.com/near/nearcore/blob/700ec29270f72f2e78a17029b4799a8228926c07/chain/network/src/network_protocol/peer.rs#L13-L19
```

Comment:
Linking just the `peer_info` here is confusing without additional context. The reader needs deeper knowledge of the network crate to understand that the `peer_info` is what is shared between nodes. I would suggest just deleting this line, the above explanation that RPC addresses are not shared is sufficient.

Reply:
Took me a few hours to figure what (and where in the code) exactly is exchanged between peer nodes is the peer info. Think this line would be helpful for anyone modifying this code in future or those trying to understand why the numbers were hacked in. 

Also difficult for future developers to know if this comment is outdated, so this line allows them to view the master branch of that file to see if this comment of what exactly is being exchanged is still up to date. Hence, leaving this line in there. 

The first page itself explains the definition of 'peer' of being other peer nodes, so understanding the context of peer info after that shouldn't be too difficult 
https://near.github.io/nearcore/architecture/index.html